10-autoencoder.ipynb

```diff
@@ -231,26 +294,33 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 232
+    },
+    "colab_type": "code",
+    "id": "H1oNdAxj2GrK",
+    "outputId": "6f30e1ae-a3d1-4fbb-9ee1-6eba91fce10b"
+   },
    "outputs": [],
    "source": [
     "# Compare the results: [noise], [img + noise], [img], [AE, Telea, Navier-Stokes] inpainting\n",
     "\n",
     "with torch.no_grad():\n",
-    "    display_images(img_bad.grad_fn.noise[3:7], img_bad[3:7])\n",
```

Comment:
This is the bug.
`img_bad.grad_fn.noise` does not exist.
Plz, revert the rest, and figure out where this tensor went.